hitbox-redis/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "hitbox-redis"
 version = "0.1.0"
 authors = ["Belousow Makc <lib.bmw@gmail.com>", "Andrey Ermilov <andrerm@ya.ru>"]
 license = "MIT"
-edition = "2021"
+edition = "2018"
```

Comment:
Really need 2018 edition?

Reply:
It didn't build with different editions in some cases(but i don't remember in which cases). I think it's better to do the opposite - update to 2021